irida_import/irida_import.xml.sample

```diff
@@ -1,11 +1,17 @@
 <tool add_galaxy_url="False" force_history_refresh="True" id="irida_import" name="IRIDA" tool_type="data_source" version="1.4.0">
```

Comment:
Should be bump up this tool version, since we're making a bunch of code changes now?

Reply:
I decided to bump to 2.0.0 as there are some breaking changes in how the tool is installed.